src/pudl/convert/ferc_to_sqlite.py

```diff
@@ -8,6 +8,7 @@
 import argparse
```

Comment:
Should we move this file to the `ferc_to_sqlite` module? Could be something like `src/pudl/ferc_to_sqlite/cli.py`

Reply:
I think that's a less surprising place for it, yeah.

Reply:
Should we move `src/pudl/cli.py` to `src/pudl/etl/cli.py`?